src/api/instances.tsx

```diff
@@ -333,3 +337,30 @@ export const fetchInstanceLogFile = (
       .catch(reject);
   });
 };
+
+export const duplicateInstance = (
+  values: LxdInstanceDuplicate,
+  instance: LxdInstance,
+): Promise<LxdOperationResponse> => {
+  return new Promise((resolve, reject) => {
+    fetch(`/1.0/instances`, {
+      method: "POST",
+      body: JSON.stringify({
+        //Does not yet consider targetClusterMember or storagePool
+        description: instance.description,
+        name: values.instanceName,
+        source: {
+          allow_inconsistent: values.allowInconsistent,
+          instance_only: values.instanceOnly,
+          project: values.project,
```

Comment:
This line here relates to the issue @piperdeck reported. Looking at the [API doc](https://documentation.ubuntu.com/lxd/en/latest/api/#/instances/instances_post), `source.project` is where the source instance is located. The target project should be specified by the `?project=` query param.

Reply:
also `source.project` won't be needed since in the UI, the instance detail page is already part of a project context